nucleus/payara-modules/payara-micro/src/main/java/fish/payara/micro/PayaraMicro.java

```diff
@@ -715,6 +868,25 @@ private void scanArgs(String[] args) {
                         throw new IllegalArgumentException();
                     }   i++;                    
                     break;
+                case "--autoBindHttp":
+                    autoBindHttp = true;
+                    break;
+                case "--autoBindSsl":
+                    autoBindSsl = true;
+                    break;
+                case "--autoBindRange":
+                    String autoBindRangeString = args[i + 1];
+                    try {
+                        autoBindRange = Integer.parseInt(autoBindRangeString);
+                        if (autoBindRange < 1) {
+                            throw new NumberFormatException("Not a valid auto bind range");
+                        }
+                    } catch (NumberFormatException nfe) {
+                        System.err.println(autoBindRangeString
```

Comment:
How do you mean? I designed it so that it doesn't fail if the input for `--autoBindRange` is incorrect. If it's incorrect, it just reverts back to its default of 5.